webapp/packages/eslint-config/flat.js

```diff
@@ -111,7 +111,7 @@ export default tseslint.config(
       ],
       '@stylistic/comma-spacing': ['error'],
       '@typescript-eslint/default-param-last': ['error'],
-      '@stylistic/func-call-spacing': ['error'],
+      '@/func-call-spacing': ['error'],
```

Comment:
Why only this rule? And this fix seems strange https://eslint.style/guide/migration